js/app.js

```diff
@@ -0,0 +1,277 @@
+// jQuery extension to handle css animations
+
+$.fn.extend({
+    animateCss: function(animationName) {
+        var animationEnd = 'webkitAnimationEnd mozAnimationEnd MSAnimationEnd oanimationend animationend';
+        $(this).addClass('animated ' + animationName).one(animationEnd, function() {
+            $(this).removeClass('animated ' + animationName);
+        });
+    }
+});
+
+
+// Pirate Bar Sounds // 
+
+var tavern = new Audio('sounds/Medieval_tavern.mp3');
+var yar = new Audio('sounds/yaargh.wav');
+var drivesMe = new Audio('sounds/drivesmenuts.wav');
+var shiver = new Audio('sounds/shivermetimbers.wav');
+var ahoy = new Audio('sounds/ahoymatey.wav');
+var lillylivered = new Audio('sounds/lillyliveredlandlubbers.wav');
+var plank = new Audio('sounds/walktheplank.wav');
+
+
+// Constructor Functions // 
+
+var PirateQuestions = function(questions, flavor) {
+    this.questions = questions;
+    this.flavor = flavor;
+
+};
+
+
+var PirateIngredients = function(ingredients, amounts) {
+    this.ingredients = ingredients;
+    this.amounts;
+};
+
+
+var PirateBartender = function(questions) {
+    this.questions = questions;
+    this.userPreference = [];
+}
+
+
+
+PirateQuestions.prototype.display = function() {   
+    $('.pirate-questions').text(' ');
+    $('.pirate-questions').animateCss('zoomIn');
+    $('.pirate-questions').append('<h2>' + this.questions + '</h2>');
+
+};
+
+
+// store random number and use same random number in splice()
+// make var for holding random num 
+
+
+
+PirateBartender.prototype.checkUserAnswerYes = function() {
+	if (this.questions.length < 0) {
```

Comment:
Refactor this so that you don't have an empty block. Here, instead of saying `if (true) {[do nothing} else {do something}` you can streamline by saying `if (not true) {do something}`